package.json

```diff
@@ -18,6 +18,7 @@
     "publish-storybook": "bash .scripts/publish_storybook.sh"
   },
   "devDependencies": {
+    "@kadira/storybook": "^1.35.1",
```

Comment:
The `@kadirahq/react-storybook-addon-info` repo also has a storybook (storyception??)